starwars_api/models.py

```diff
@@ -11,15 +11,21 @@ def __init__(self, json_data):
         Dynamically assign all attributes in `json_data` as instance
         attributes of the Model.
         """
-        pass
+        for key, value in json_data.items():
+            setattr(self, key, value)
+            # setattr(x, 'foobar', 123) is equivalent to x.foobar = 123
```

Comment:
There is a getattr just as useful which allows to try to get an attribute and has a safe default value to assign.